flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/plan/utils/JoinUtil.scala

```diff
@@ -50,6 +52,14 @@ import scala.collection.JavaConversions._
 /** Util for [[Join]]s. */
 object JoinUtil {
 
+  /** Returns true if mini-batch for join is enabled. */
+  def isMiniBatchEnabled(config: ReadableConfig): Boolean = {
+    val enableMiniBatch = config.get(ExecutionConfigOptions.TABLE_EXEC_MINIBATCH_ENABLED)
+    val miniBatchLatency =
```

Comment:
We can remove this check because `StreamCommonSubGraphBasedOptimizer#doOptimize` method had checked it, And we can remove this method.